pkg/symlink/fs.go

```diff
@@ -16,11 +16,11 @@ import (
 
 // FollowSymlinkInScope is a wrapper around evalSymlinksInScope that returns an absolute path
 func FollowSymlinkInScope(path, root string) (string, error) {
```

Comment:
@jhowardmsft update function comment to explain that it handles slash-separated generic paths.